config/crd/bases/v1/datadoghq.com_datadogagents.yaml

```diff
@@ -6575,6 +6575,161 @@ spec:
                               More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
                             type: object
                         type: object
+                      securityContext:
+                        description: Pod-level SecurityContext.
+                        properties:
+                          fsGroup:
+                            description: "A special supplemental group that applies
+                              to all containers in a pod. Some volume types allow
+                              the Kubelet to change the ownership of that volume to
+                              be owned by the pod: \n 1. The owning GID will be the
+                              FSGroup 2. The setgid bit is set (new files created
+                              in the volume will be owned by FSGroup) 3. The permission
+                              bits are OR'd with rw-rw---- \n If unset, the Kubelet
+                              will not modify the ownership and permissions of any
+                              volume."
+                            format: int64
+                            type: integer
+                          fsGroupChangePolicy:
+                            description: 'fsGroupChangePolicy defines behavior of
+                              changing ownership and permission of the volume before
+                              being exposed inside Pod. This field will only apply
+                              to volume types which support fsGroup based ownership(and
```

Comment:
```suggestion
                              to volume types which support fsGroup based ownership (and
```

Reply:
Hello May,
Unfortunately, the whole [config/crd/bases/v1/datadoghq.com_datadogagents.yaml](https://github.com/DataDog/datadog-operator/pull/448/files/c8260fcc205aaae04209420e3b39f4792c2d708c#diff-c345ff9030e958ce4055ba0bfd336001d9801c7273b1775131e727c3c6e04f9e) file is fully generated and the original source from which all those descriptions are coming from is not owned by Datadog.

For ex., this suggestion concerns this text: https://github.com/kubernetes/api/blob/v0.23.3/core/v1/types.go#L3496 (not from Datadog org)

Reply:
Ah, okay, thanks for letting me know!